src/test/java/ro/unibuc/triplea/application/games/steam/web/SteamGameControllerTestIT.java

```diff
@@ -0,0 +1,58 @@
+package ro.unibuc.triplea.application.games.steam.web;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import ro.unibuc.triplea.application.games.steam.dto.response.SteamGameResponse;
+import ro.unibuc.triplea.domain.games.steam.service.SteamGameService;
+import java.util.Optional;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.when;
+
+@SpringBootTest
+@Tag("IT")
+public class SteamGameControllerTestIT {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private SteamGameService steamGameService;

```

Comment:
do not use @Autowired in tests. class that wants to be tested should be instantiated and use mock parameters.

Reply:
As I mentioned above this class is a leftover from another branch, the test here will be done by @alexbanilean . I can remove it from this PR to avoid confusion, and it will be added back when it is complete